### PR TITLE
ENH add pixmappy

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pixmappy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gbernstein/pixmappy/archive/v{{ version }}.tar.gz
+  sha256: 9ef1ddeaea58ca50cbf7226a2c68ac183726ae3e97028f9011849dd086f7c48d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (beckermr) [inline recipes/pixmappy/meta.yaml:14]:
```suggestion
  noarch: python
  script: "{{ PYTHON }} -m pip install . -vv"
```

**Comment by beckermr** [comment]
@conda-forge/staged-recipes this one is ready for review!

It is a `noarch: python` package and the LICENSE file is in the upstream repo but not yet in the latest release.

File: recipes/pixmappy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+test:
+  imports:
+    - pixmappy
+
+about:
+  home: https://github.com/gbernstein/pixmappy
+  license: BSD-2-Clause
+  license_family: BSD
+  license_file: LICENSE
```
Comment (chrisburr) [inline recipes/pixmappy/meta.yaml:38]: I think you need to include this as well: https://github.com/gbernstein/pixmappy/blob/master/Piff_LICENSE

Comment (beckermr) [reply recipes/pixmappy/meta.yaml:38]: Good catch!

Comment (beckermr) [reply recipes/pixmappy/meta.yaml]: done!

**Comment by ReimarBauer** [comment]
seems it misses on windows
conda.exceptions.ResolvePackageNotFound: 
  - lsstdesc.coord

**Comment by beckermr** [comment]
Yup. This is a noarch python recipe. That means it only needs to build on linux. It will install on windows if anyone ever ships all of its deps on windows.

**Comment by beckermr** [comment]
Thanks @chrisburr!

**Comment by ReimarBauer** [comment]
only for documentation, there is not a description for windows on that page.
https://github.com/LSSTDESC/Coord

**Pull request merged**